tox.ini

```diff
@@ -13,7 +13,7 @@ commands =
 deps =
     pycodestyle
 commands =
-    pycodestyle flask_debugtoolbar test
+    pycodestyle src/flask_debugtoolbar test --ignore=E731,W503,W504
```

Comment:
this would be nicer to annotate with the reasoning for adding these ignores in a separate PR. No need for a code comment on them, but at least a PR comment woudl be nice.

Reply:
Don't forget to open a PR for this if you think it's useful!

Reply:
Created https://github.com/pallets-eco/flask-debugtoolbar/pull/219